packages/wallet-manager/src/vaults/mnemonic-vault.ts

```diff
@@ -56,7 +51,7 @@ export class MnemonicVault implements Vault<MnemonicVaultOptions> {
     do {
       const wallet = Wallet.fromMnemonic(
         this.#secret,
-        this.provider,
+        undefined,
         this.getDerivePath(numberOfAccounts)
```

Comment:
Maybe we should change the order of params on `fromMnemonic`?

Reply:
Good catch! 🎯